content/post/data-dies-in-darkness.md

```diff
@@ -0,0 +1,73 @@
+
+For example, maybe it's someone's job to once-a-month report out on distribution costs by product mix. Because the systems weren't designed to be integrated, this involves some fairly tedious excel work (download some stuff from system x, find the email from person y, index-match and pivot). In this work-flow, if there's any data issues, they are quickly caught by the person doing this reporting work ("that doesn't look right, I need to check z.") 
+
+If you can automate this workflow, you can automate the data quality checks, and rather than having a person doing quality checks on their local lap-top, you can get them checking the warehouse for quality (where everyone benefits). For example, maybe you can scrape the report from system z, automatically capture excel reports sent via email, and perform the remaining joins and summaries in SQL.
+
+Now, not only do you have these data in the warehouse available for other uses, but you also have another recruit who is helping you monitor data quality.
+
+#### Dashboard everything, and send emailed reports (including relevant executives) at a regular cadence
+It should be easy for anyone in the business to find and explore any piece of data that is interesting to them. This means building lots of dashboards that are continuously monitoring the business's KPIs. Sunlight really is the best disinfectant: you'll catch data quality issues much more quickly if you have the whole company monitoring the data on an ongoing basis. You should encourage people to reach out with questions if they see something that looks wrong.
+
+In addition to simply making the dashboards available (and ideally searchable!) you should try to schedule regular email reports that go out to wide audiences. If the people operationally responsible for the metrics in the dashboard know that it's going to be sent to their boss on Tuesdays, they'll be extra vigilant when it comes to making sure the data reflected there-in are accurate and up-to-date (so they don't have to field an email from their boss asking "WTF?").
+
+#### Make heavy use of "data quality" alerts and monitoring
+Finally, you should develop tooling for programatically alerting you to data quality issues. If you rely on a "mapping" table for data quality (e.g., SKUs to prices) then you should set up an alert if there are rows missing in that table ("ALERT: SKUs found in warehouse without a corresponding mapping"). Similarly, you can check when the "rate" of something dips above or below a certain threshold ("ALERT: more than 10% of orders are missing corresponding UTM tagging information.") This allows your team to pro-actively identify and monitor potentiall issues rather than being caught off guard when a business partner sees some crazy looking data three weeks later.
```

Comment:
I'm a big fan of this. maybe also 
 - when you catch a problem and correct it, notify the stakeholders in case they also saw something funny and didn't mention it, and to demonstrate the pro-active monitoring to build trust